pkg/config/v2/parameter/value/value.go

```diff
@@ -0,0 +1,90 @@
+// @license
+// Copyright 2021 Dynatrace LLC
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package value
+
+import (
+	"github.com/dynatrace-oss/dynatrace-monitoring-as-code/pkg/config/v2/errors"
+	"github.com/dynatrace-oss/dynatrace-monitoring-as-code/pkg/config/v2/parameter"
+)
+
+// type of the parameter used in config files
+const VALUE_PARAMETER_TYPE = "value"
+
+var VALUE_PARAMETER_SERDE = parameter.ParameterSerDe{
+	Serializer:   writeValueParameter,
+	Deserializer: parseValueParameter,
+}
+
+// parameter representing a simple value. the value has to be
+// resolve at config load time.
+type ValueParameter struct {
+	Value interface{}
+}
+
+// this forces the compiler to check if ValueParameter is of type Parameter
+var _ parameter.Parameter = (*ValueParameter)(nil)
+
+func (p *ValueParameter) GetType() string {
+	return VALUE_PARAMETER_TYPE
+}
+
+func (p *ValueParameter) GetReferences() []parameter.ParameterReference {
+	// the value parameter cannot have references, as it is a simple value
+	return []parameter.ParameterReference{}
+}
+
+func (p *ValueParameter) ResolveValue(context parameter.ResolveContext) (interface{}, error) {
```

Comment:
Unused param
```suggestion
func (p *ValueParameter) ResolveValue() (interface{}, error) {
```

Reply:
This is required by the interface, so we cannot get rid of this parameter.